translations/messages.fr.xlf

```diff
@@ -0,0 +1,3065 @@
+        The owner of a stack is the one that has a piece at the top of the stack.
+        Only the owner of a stack can move it.
+        A stack cannot be moved if it is surrounded by 6 other stacks.
+        A stack is moved by as many spaces as there are pieces within, in a straight line, and it should land on an occupied space.
+        This line cannot move along the vertex of two neighboring spaces, as would a vertical move.
+        Therefore, there are six possible directions.
+        The player with dark pieces starts.&lt;br/&gt;&lt;br/&gt;
+        You&apos;re playing Dark, click on a stack and move it by one space.</source>
+        <target>Au Dvonn, chaque case hexagonale comporte une pile de pièce.
+        Si aucun nombre n&apos;est indiqué sur une pile, c&apos;est qu&apos;elle ne comporte qu&apos;une pièce.
+        Le nombre écrit sur une pile correspond au nombre de pièces empilées et donc le nombre de point qu’elle rapporte à son propriétaire.
+        Son propriétaire est celui dont une pièce est au sommet de la pile.
+        Seul son propriétaire peut déplacer la pile.
+        Il ne peut pas la déplacer si elle est entourée par 6 autres piles.
+        Il la déplace d’autant de cases que sa hauteur, en ligne droite, et doit atterrir sur une case occupée.
+        Cette ligne droite ne peut pas passer le long de l&apos;arête de deux cases voisines, comme le ferait un déplacement vertical.
+        Il y a donc six directions possibles.
+        Le joueur avec les piles foncées commence.&lt;br/&gt;&lt;br/&gt;
+        Vous jouez avec Foncé, cliquez sur une pile puis déplacez la d&apos;une case.</target>
+      </trans-unit>
+      <trans-unit id="8769382369391878948" datatype="html">
+        <source>Disconnection</source>
+        <target>Déconnection</target>
+      </trans-unit>
+      <trans-unit id="90245733925298447" datatype="html">
+        <source>Red pieces are called &quot;sources&quot;.
+        When a stack is not directly nor indirectly connected to a source, it is removed from the board.
+        You can move on a source: this forms a stack that contains the source, and which will never be disconnected.
+        Besides this, a stack containing a source acts like any other stack: you can move it, and your opponent can move on that stack to take possession of it.&lt;br/&gt;&lt;br/&gt;
+        You&apos;re playing Dark, move your piece on the source.</source>
+        <target>Les pièces rouges sont appelées “sources”.
+        Quand une pile n’est plus directement ou indirectement connectée à une source, elle est enlevée du plateau.&lt;br/&gt;&lt;br/&gt;
+        Vous jouez Foncé, déplacez votre pièce sur la source.</target>
+      </trans-unit>
+      <trans-unit id="1204343429877937229" datatype="html">
+        <source>Nice, you have disconnected 4 pieces of your opponent!
+        He has lost 4 points.
```

Comment:
Did you just assume my gender :offended:

Reply:
:fearful: